packages/x-license-pro/bin/license-decode-script.js

```diff
@@ -2,4 +2,8 @@
 /* eslint-disable */
 require = require('esm')(module);
 
-require('../build/node/cli/license-cli').licenseDecodeCli();
+if (process.env.NODE_ENV === 'dev') {
+  require('../build/node/cli/license-cli').licenseDecodeCli();
```

Comment:
What's the use case for this? Do you use the CLI through npx in our local repository?
This will break package CLI in environment with `NODE_ENV=dev`